restli-int-test/src/test/java/com/linkedin/restli/examples/TestHttp2With204AndException.java

```diff
@@ -0,0 +1,55 @@
+package com.linkedin.restli.examples;
+
+import com.linkedin.r2.transport.http.client.HttpClientFactory;
+import com.linkedin.r2.transport.http.common.HttpProtocolVersion;
+import com.linkedin.restli.client.Request;
+import com.linkedin.restli.client.Response;
+import com.linkedin.restli.examples.greetings.api.Greeting;
+import com.linkedin.restli.examples.greetings.client.GreetingsBuilders;
+import com.linkedin.restli.server.RestLiConfig;
+import com.linkedin.restli.test.util.RootBuilderWrapper;
+import java.util.HashMap;
+import java.util.Map;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+
+public class TestHttp2With204AndException extends RestLiIntegrationTest{
```

Comment:
The issue happens with Jetty + HTTP/2.
However, I'm not sure if it happens with the Test Server + HTTP/2. If not, writing tests separately for HTTP1 and HTTP2 doesn't make much sense. The only thing we need to test IMHO is the response that we are trying to send to the client, with an empty body for this case. Nothing more.

Reply:
The test server actually uses http jetty server provided by r2. 
It might not reproduce the failure if the client behavior is different in the test suite. It does add some value as it guards against change in behavior within the protocol (say jetty HTTP 2.0 decides to add some response body if error response header is true). I'll let you decide if you want to keep it.